README.md

```diff
@@ -126,7 +138,58 @@ const sendBundleResponse = await skipBundleClient.sendBundle(signedBundle, DESIR
 ```
 
 `DESIRED_HEIGHT_FOR_BUNDLE` should be a number, where 0 asks the Sentinel to autodetermine the next height.
+### Terra and others using terra.js / feather.js
+Terra and [alliance](https://github.com/terra-money/alliance) chains usually use an alternative client library called [terra.js](https://github.com/terra-money/terra.js), it was rebranded to [feather.js](https://github.com/terra-money/feather.js) in late 2022 to add multichain support (i.e. similar to cosmjs, you can use it to interact with not only terra but also other cosmos SDK chains, most likely alliance chains that share security with Terra).
+
+The overall logic is similar to using cosmjs. Modified from the [send native token example](https://docs.terra.money/develop/feather-js/common-examples#sending-native-tokens).
+
+Import skip and feather.
+```
+import { SkipBundleClient } from '@skip-mev/skipjs'
+import { LCDClient, MnemonicKey, MsgSend } from '@terra-money/feather.js'
+```
 
+Create LCD and get mnemonic key.
+```
+const lcd = new LCDClient({
+    URL: LCD_ENDPOINT,
+    chainID: CHAIN_ID,
+})
+
+const mk = new MnemonicKey({
+  mnemonic:'', // Insert your mnemonic
+})
+const wallet = lcd.wallet(mk)
+const privKey = mk.privateKey
+```
+
+Construct message of send token to your contact address and another send token to [skip auction house address](https://docs.skip.money/searcher#winning-the-auction).
+```
+const sendToYourContact = new MsgSend(
+  wallet.key.accAddress('terra'), // requires prefix as a parameter
+  'terra1dcegyrekltswvyy0xy69ydgxn9x8x32zdtapd8',
```

Comment:
Let's not include any hardcoded addresses here. Can we just replace any hardcoded addresses with understandable const names? e.g. `TO_ADDRESS` or something similar?